src/igvfd/schemas/changelogs/human_donor.md

```diff
@@ -4,6 +4,7 @@
 
 * *taxon_id* now restricted to NCBI taxonomy ids that start with NCBI:txid followed by numbers. Example NCBI:txid9606 for *Homo sapiens*
 * *taxon_id* replaced by organism
-* defined *sex* as required property and defaulted it to *unspecified*
-* human_donor has aliases mixin and submitted mixin removed, but still has properties for aliases and submitted due to change of basic_item mixin.  The actual properties have not changed.
-* organism property renamed to taxa
+* Defined *sex* as required property and defaulted it to *unspecified*
+* *human_donor* has *aliases* mixin and *submitted* mixin removed, but still has properties for *aliases* and *submitted* due to change of basic_item mixin.  The actual properties have not changed.
+* *organism* property renamed to *taxa*
+* Donors now have a property *traits* to specify known phenotypic traits
```

Comment:
Meenakshi, while it may seem like a good idea I don't think updating the changelog for properties unrelated to the ticket should be done. Additionally, here it should be Human donor rather than Donor.

Reply:
Okay.

Reply:
Still not Human Donors, but Donors, please correct